packages/plugin-seo/src/index.tsx

```diff
@@ -12,12 +12,18 @@ import type {
   SEOPluginConfig,
 } from './types.js'
 
-import { MetaDescription } from './fields/MetaDescription.js'
-import { MetaImage } from './fields/MetaImage.js'
-import { MetaTitle } from './fields/MetaTitle.js'
+import { MetaDescriptionComponent } from './fields/MetaDescription/MetaDescriptionComponent.js'
```

Comment:
Might want to avoid barrel files for this. Can we just expose a `/fields` export?

Reply:
moved into a `fields` file in exports